chain/substreams/src/chain.rs

```diff
@@ -13,42 +22,25 @@ use graph::{
     impl_slog_value,
     prelude::{async_trait, BlockNumber, ChainStore},
     slog::Logger,
+    substreams,
 };
+use prost::Message;
 
-use crate::{data_source::*, TriggerData, TriggerFilter, TriggersAdapter};
+use crate::{
+    data_source::*, EntitiesChanges, SubstreamBlock, TriggerData, TriggerFilter, TriggersAdapter,
+};
 
 #[derive(Clone, Debug)]
 pub struct Block {}
 
-impl blockchain::Block for Block {
-    fn ptr(&self) -> BlockPtr {
-        todo!()
-    }
-
-    fn parent_ptr(&self) -> Option<BlockPtr> {
-        todo!()
-    }
-
-    fn number(&self) -> i32 {
-        self.ptr().number
-    }
-
-    fn hash(&self) -> blockchain::BlockHash {
-        self.ptr().hash
-    }
-
-    fn parent_hash(&self) -> Option<blockchain::BlockHash> {
-        self.parent_ptr().map(|ptr| ptr.hash)
-    }
-
-    fn data(&self) -> Result<jsonrpc_core::serde_json::Value, jsonrpc_core::serde_json::Error> {
-        Ok(jsonrpc_core::serde_json::Value::Null)
-    }
-}
-
 #[derive(Debug)]
 pub struct Chain {}
 
+#[derive(Clone, Debug)]
+pub enum GraphEntityFinality {
+    // Final(Arc<>)
```

Comment:
Was this supposed to have some content?

Reply:
This has been removed, thanks for the heads up